android/src/main/kotlin/live/hms/hmssdk_flutter/methods/HMSPipAction.kt

```diff
@@ -13,6 +13,8 @@ class HMSPipAction {
 
     companion object {
         var pipResult: Result? = null
+        private var pipAutoEnterEnabled = false
```

Comment:
is the default autoEnterPIP value true or false?
in the code comments below, it's mentioned as `true`